rollup.config.js

```diff
@@ -95,39 +96,8 @@ export default {
       return name.replace(/^Sources[/\\]/, '');
     },
   },
-  external: [/@babel\/runtime/],
+  external: Object.keys(packageJSON.dependencies).map((name) => new RegExp(`^${name}`)),
```

Comment:
There's a [`rollup-plugin-auto-external`](https://www.npmjs.com/package/rollup-plugin-auto-external) that should cover more cases (such as transitive dependencies)

Reply:
I'll check it out. Thanks for the suggestion!